front/src/applications/stdcm/components/StdcmForm/StdcmOperationalPoint.tsx

```diff
@@ -152,7 +152,7 @@ const StdcmOperationalPoint = ({
   };
 
   useEffect(() => {
-    if (location) {
+    if (location && location.name) {
```

Comment:
`name` always exists in `location`
```suggestion
    if (location) {
```

Reply:
done